test/grouping.jl

```diff
@@ -904,8 +905,11 @@ Base.isless(::TestType, ::TestType) = false
 @testset "combine with aggregation functions (skipmissing=$skip, sort=$sort, indices=$indices)" for
     skip in (false, true), sort in (false, true), indices in (false, true)
     Random.seed!(1)
-    df = DataFrame(a = rand([1:5;missing], 20), x1 = rand(1:100, 20),
-                   x2 = rand(1:100, 20) +im*rand(1:100, 20))
+    # 5 is there to ensure we test a single-row group
+    df = DataFrame(a = [rand([1:4;missing], 19); 5],
+                   x1 = rand(1:100, 20),
+                   x2 = rand(1:100, 20) + im*rand(1:100, 20),
+                   x4 = rand(1:100, 20) .* u"m")
```

Comment:
can we also add `x5` column that would be like `x4` but also contain `missing` value and then below, in particular we would test also `var∘skipmissing` etc. (and make sure that we have a group that has only `missing` values in it).

In general I would want to test what happens if fast aggregation functions get a vector of zero length (which is only possible with `missing`s as currently our groups must have a positive length).

Reply:
This is tested below. Do you mean we should also test it for Unitful? I'm afraid that will require duplicating the whole block, as `prod` doesn't apply to these so we can't just loop over different types of columns. And then there's also `Complex` and `CategoricalArray`...

Reply:
Yes - I meant a test with `Unitful`. Maybe then just add a small test with e.g. `var∘skipmissing` on a concrete test DataFrame like: one group only, column is union of missing and `Unitful` and contains only `missing` values?

Reply:
OK, let's bite the bullet and add systematic tests like for floats. I've pushed a commit.